src/importers/azureDevops.js

```diff
@@ -45,6 +45,12 @@ async function sendGetHttpRequest(endpoint, forWhat) {
         value: organization.accountName,
       }));
       return organizationInfo;
+    case "projectInfo":
```

Comment:
This pattern isn't ideal. Should we refactor this now?

Reply:
It does seem a little odd. Normally there would just be a simpler `get` method to execute the request itself, and then light wrapper methods like `getUser`, etc.

I see methods like that exist below, I would expect these transformations to occur in those methods, not here.

Reply:
Agreed. I'll revise this.